app/models/lecture.rb

```diff
@@ -830,12 +830,6 @@ def speakers
     User.where(id: SpeakerTalkJoin.where(talk: talks).select(:speaker_id))
   end
 
-  def older_than?(timespan)
-    return true unless term
-
-    term.begin_date <= Term.active.begin_date - timespan
-  end
-
   def stale?
```

Comment:
Maybe one last thing: could you add a docstring including an explanation for what "stale" means? (doesn't have to include every detail, just a high-level overview maybe)